src/components/admin-ui/AddYachts.js

```diff
@@ -31,27 +39,33 @@ function AddYachts() {
   };
 
   return (
-    <Form onSubmit={handleSubmit}>
-      <Form.Group className="mb-3" controlId="formAddYacht">
-        <Form.Label>Yacht Name</Form.Label>
-        <Form.Control type="text" htmlFor="name" name="name" placeholder="Enter Yacht Name" onChange={handleChange} />
-      </Form.Group>
-      <Form.Group className="mb-3" controlId="formAddYacht">
-        <Form.Label>Description</Form.Label>
-        <Form.Control type="textarea" htmlFor="descripion" name="description" placeholder="Enter Yacht Description" onChange={handleChange} />
-      </Form.Group>
-      <Form.Group className="mb-3" controlId="formAddYacht">
-        <Form.Label>Price</Form.Label>
-        <Form.Control type="number" htmlFor="price" name="price" placeholder="Enter Yacht Price" onChange={handleChange} />
-      </Form.Group>
-      <Form.Group className="mb-3" controlId="formAddYacht">
-        <Form.Label>Image</Form.Label>
-        <Form.Control type="file" id="image" name="image" className="image" placeholder="Image" />
-      </Form.Group>
-      <Button variant="primary" type="submit">
-        ADD YACHT
-      </Button>
-    </Form>
+    <main className="main">
+      <div className="effect" />
+      <div className="showcase">
+        <div className="logo"><img src={logo} alt="logo" /></div>
+        <Form onSubmit={handleSubmit} className="bg-light rounded-3 shadow p-4 bg-body">
+          <Form.Group className="mb-3" controlId="formAddYacht">
+            <Form.Label>Yacht Name</Form.Label>
+            <Form.Control type="text" htmlFor="name" name="name" placeholder="Enter Yacht Name" onChange={handleChange} />
+          </Form.Group>
+          <Form.Group className="mb-3" controlId="formAddYacht">
+            <Form.Label>Description</Form.Label>
+            <Form.Control type="textarea" htmlFor="descripion" name="description" placeholder="Enter Yacht Description" onChange={handleChange} />
+          </Form.Group>
+          <Form.Group className="mb-3" controlId="formAddYacht">
+            <Form.Label>Price</Form.Label>
+            <Form.Control type="number" htmlFor="price" name="price" placeholder="Enter Yacht Price" onChange={handleChange} />
+          </Form.Group>
+          <Form.Group className="mb-3" controlId="formAddYacht">
+            <Form.Label>Image</Form.Label>
+            <input type="file" id="image_url" name="image_url" placeholder="Select Image" />
+          </Form.Group>
+          <Button variant="primary" type="submit">
+            ADD YACHT
+          </Button>
+        </Form>
+      </div>
+    </main>
```

Comment:
- [ ] You did a great job, however, when you add a yacht it is added to the db but it's not displaying any message. I recommend fixing this issue.